### PR TITLE
[RA1 -all Ch] Proofreading

### LINKED ISSUE #2466: [WS-All Chxx] Proofreading
RM
- Ch02 BNG should Broadband Network Gateway (not Border)
- Ch07: 7.4.3 Platform ‘front-end’ access security      DDOS should be DDoS
- Ch10: 10.3.5 link in TOC
- Ch10: make Cyborg Project active link rather than have url separately listed
- All - change "Network Intensive" to "High Performance"

RA1
- Ch 2 IaaS, PaaS, SaaS: capitalisation error

RC1
- Ch02: VFNSDK -- should be VNF SDK

### PR DESCRIPTION
**Pull request by karinesevilla**
Partially Fixes Issue #2466

### CODE REVIEW

**Comment by karinesevilla** [comment]
@pgoyal01 @walterkozlowski @collivier @ildikov @ulikleber 
Please review, Tuesday 15, last day to merge proofreading

File: doc/ref_arch/openstack/chapters/chapter01.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [<< Back](../../openstack)
 
 # 1. Overview
-<p align="right"><img src="../figures/bogo_dfp.png" alt="Dickering over Fine Points" title="Bogo: Dickering over Fine Points" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_com.png" alt="Dickering over Fine Points" title="Bogo: Dickering over Fine Points" width="35%"/></p>
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.md:4]:
```suggestion
<p align="right"><img src="../figures/bogo_com.png" alt="Complete" title="Bogo: Complete" width="35%"/></p>
```

File: doc/ref_arch/openstack/chapters/chapter03.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [<< Back](../../openstack)
 
 # 3. Cloud Infrastructure Architecture - OpenStack
-<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_com.png" alt="scope" title="Scope" width="35%"/></p>
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter03.md:4]:
```suggestion
<p align="right"><img src="../figures/bogo_com.png" alt="Complete" title="Complete" width="35%"/></p>
```

File: doc/ref_arch/openstack/chapters/chapter04.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [<< Back](../../openstack)
 
 # 4. Cloud Infrastructure + VIM Component Level Architecture
-<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_com.png" alt="scope" title="Scope" width="35%"/></p>
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter04.md:4]:
```suggestion
<p align="right"><img src="../figures/bogo_com.png" alt="Complete" title="Complete" width="35%"/></p>
```

File: doc/ref_arch/openstack/chapters/chapter05.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [<< Back](../../openstack)
 
 # 5. Interfaces and APIs
-<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_com.png" alt="scope" title="Scope" width="35%"/></p>
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter05.md:4]:
```suggestion
<p align="right"><img src="../figures/bogo_com.png" alt="Complete" title="Complete" width="35%"/></p>
```

File: doc/ref_arch/openstack/chapters/chapter08.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 [<< Back](../../openstack)
 
 # 8. Gaps, Innovation, and Development
-<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_com.png" alt="scope" title="Scope" width="35%"/></p>
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.md:4]: @karinesevilla Will Chapter 8 be ever complete as it is about Gaps, Innovation, etc.? Maybe "Dickering over Fine Points"?
Depending upon what you decide, the "alt" and "title" will also need to be adjusted.

File: doc/ref_arch/openstack/chapters/chapter03.md (hunk 2 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter03.md:4]: @pgoyal01 Suggestion integrated

File: doc/ref_arch/openstack/chapters/chapter04.md (hunk 3 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter04.md:4]: @pgoyal01 Suggestion integrated

File: doc/ref_arch/openstack/chapters/chapter05.md (hunk 4 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter05.md:4]: @pgoyal01 Suggestion integrated

File: doc/ref_arch/openstack/chapters/chapter08.md (hunk 5 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter08.md:4]: @pgoyal01 Suggestion integrated

Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter08.md]: Move to dickering over fine points

**Review by walterkozlowski — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Comment by pgoyal01** [comment]
@karinesevilla Can you please resolve conflicts? Thanks

**Comment by pgoyal01** [comment]
@ildikov @ulikleber This PR is for minor proof-reading changes. Can you please review? Today is the last day for approval.

**Review by collivier — APPROVED** [review]

**Comment by karinesevilla** [comment]
@pgoyal01 @collivier Conflicts resolved

**Review by pgoyal01 — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Review by scottsteinbrueck — APPROVED** [review]

**Pull request merged**